packages/next-app/src/lib/wagmi/index.ts

```diff
@@ -12,4 +10,11 @@ export const wagmiClient = createClient({
 
 export const injectedConnector = new InjectedConnector();
 
-// export const walletConnectConnector = new WalletConnectConnector({});
+const chains = defaultChains;
```

Comment:
Is this config ok?

We are using own network configs, and I'm not sure wallet connects support all of chain we supports.

I prefer to get chains from configs/networks.json and set this wallet connect config

```
Object.entries(networks).map(([chainId, {rpc}])=>{
// create wallet connect config using info from network config
})
```